feature/google/src/main/java/com/revenuecat/purchases/google/paymentTransactionConversions.kt

```diff
@@ -14,7 +13,8 @@ fun Purchase.toStoreTransaction(
     presentedOfferingIdentifier: String?
 ): StoreTransaction = StoreTransaction(
     orderId = this.orderId,
-    skus = this.listOfSkus,
+    skus = this.products,
```

Comment:
I think we can remove `skus` from the constructor? But keep it as a deprecated property so folks that are still accessing it can easily update to `productIds`.

Reply:
Yup I think that would be much better so it's easier to remove in the future if needed. This also made me think we are making a breaking change in the constructor of this class, which is public, but devs shouldn't be using that in production code at all (maybe in tests) so it might be ok?

Reply:
since it's the same type it'll only break if they name the parameters right? 

either way, i added the constructor to the api testers... i think when we are done, comparing the api testers is how we'll see all the public changes. but maybe to be safe we should start a running changelog or something, and every PR with breaking changes should outline what's changed?

Reply:
Yeah, checking the api testers should definitely cover all api changes, but might be difficult to track what's been deprecated there, so having a changelog might be useful.

Reply:
Changelog is a great idea, I remember in the last major it was very hard to go back and take notes of all public changes